### PR TITLE
(MAINT) Adding spec test for r10k

### PR DESCRIPTION
**Pull request by MWilsonPuppet**
Adding spec test to check that the r10k gem is installed.Also changing syntax for symbols in array to use %i to get travis passing again .

### CODE REVIEW

File: puppetserver-standalone/spec/dockerfile_spec.rb [hunk 1]
```diff
@@ -24,6 +24,15 @@
     it { should be_executable }
   end
 
+  describe package('r10k') do
+    it { is_expected.to be_installed }
```
Comment (garethr) [inline puppetserver-standalone/spec/dockerfile_spec.rb]: This fails in travis because package assumes a system package, where as r10k is a gem. You can look for system gems with `by('gem')` see http://serverspec.org/resource_types.html#package

But that won't I don't think check the Puppet ruby path. You might find a suitable matcher but otherwise I would remove this. The presence of the binary is a useful check on it's own I think.

File: .rubocop.yml [hunk 2]
```diff
@@ -3,3 +3,6 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
+Style/SymbolArray:
+  Enabled: false
```
Comment (ahenroid) [inline .rubocop.yml]: @MWilsonPuppet, rather than disabling this check, do you just want to fix the Rakefile? e.g.

-    task publish: [
-      :push,
-      :update_microbadger
+    task publish: %i[
+      push
+      update_microbadger

Comment (MWilsonPuppet) [inline .rubocop.yml]: @ahenroid Hi Andy I was going to do this in a separate PR after this one but more than happy to do in this one. Ill make the changes to the rakefile now. Cheers Mark.

File: .rubocop.yml [hunk 3]
```diff
@@ -3,3 +3,4 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
```
Comment (ahenroid) [inline .rubocop.yml]: @MWilsonPuppet, do you want to remove this extra newline? After that, this looks good to me.

Comment (MWilsonPuppet) [inline .rubocop.yml]: @ahenroid Ahh sorry Andy didnt notice that. Yes Ill remove that now. cheers Mark.

**Comment by gregohardy** [comment]
+1 good job guys. I'll merge this? If you need any changes andy just yell at me :)
